app/src/main/res/drawable/ic_quest_ferry.xml

```diff
@@ -69,7 +69,7 @@
         android:fillColor="#ccd6dd"/>
     <path
         android:pathData="m1.19,23.38c1.36,3.39 3.82,6.36 7.23,8.33 2.05,1.19 4.25,1.89 6.47,2.15l3.99,0c5.12,-0.61 9.9,-3.54 12.68,-8.35 0.4,-0.69 0.74,-1.41 1.04,-2.13 -2.65,-0.52 -5.22,-0.42 -7.85,0 -2.68,0.43 -5.72,0.57 -7.85,0 -4.76,-1.28 -6.06,0.51 -7.85,0C7.55,22.95 5.18,20.92 4.51,23.27Z"
-        android:fillColor="#529add"/>
+        android:fillColor="#3e74c9"/>
```

Comment:
The car one should have the yellow background, like for the maxspeed quest etc.
The pedestrian one should be the same but have the blue background like for the pedestrian crossing quest etc

Reply:
I am pretty sure that this is a change of water color to keep it consistent - see https://github.com/westnordost/StreetComplete/pull/1432/commits/04d1d0b2370194e816c512cefcb5eeebd6ec0c78

> to make water distinguishable from blue background, and keep water color the same on both
> I made water color a bit darker 

I copied blue background from other pedestrian quest for pedestrian variant and it turned out to be the same shade as water.

So I made water darker on both icons.

Reply:
Ah okay, the diff looked as if you changed that background color. Then, all is fine, thank you!